end_to_end/5-pipeline-e2e.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>remove this "pipeline" link from all of the titles.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
need link back to the top